### PR TITLE
Edit manual kots CLI install steps

### PR DESCRIPTION
**Pull request by paigecalvert**
https://app.shortcut.com/replicated/story/67365/kots-cli-docs-are-unclear-for-mac

Preview: https://deploy-preview-929--replicated-docs.netlify.app/reference/kots-cli-getting-started#manually-download-and-install

### CODE REVIEW

File: docs/reference/kots-cli-getting-started.md [hunk 1]
```diff
@@ -89,28 +89,69 @@ Users in air gap environments can also follow this procedure to install the kots
...
+      ```
+
+1. Rename the `kots` executable to `kubectl-kots` and move it to a directory that is on your PATH. Run one of the following commands, depending on if you have write access to the target directory:
+
+   * **You have write access to the directory**:
+
+     ```bash
+     mv kots /PATH_TO_KOTS/kubectl-kots
+     ```
+     Replace `PATH_TO_KOTS` with the path to the directory. For example, `/usr/local/bin`.
```
Comment (dexhorthy) [inline docs/reference/kots-cli-getting-started.md:141]: I'm not sure this is the best placeholder text for the content it's trying to convey. I would think just inlining `/usr/local/bin` would be as useful if not more useful. When i read `PATH_TO_KOTS`, I think "oh i need to input a path where KOTS things are already located", but really, I need to put KOTS in a directory that's on my path

Comment (dexhorthy) [reply docs/reference/kots-cli-getting-started.md:141]: alternatively could use `SOME_DIRECTORY_ON_YOUR_PATH` - would be more clear

Comment (paigecalvert) [reply docs/reference/kots-cli-getting-started.md]: That's a great call

Comment (paigecalvert) [reply docs/reference/kots-cli-getting-started.md]: Also hi Dex 😂

**Review by ousbots — DISMISSED** [review]

**Review by jonquil2002 — DISMISSED** [review]
lgtm - Fantastic! Just one comment - I think there's a straggler in there.

File: docs/reference/kots-cli-getting-started.md [hunk 2]
```diff
@@ -89,28 +89,75 @@ Users in air gap environments can also follow this procedure to install the kots
...
+
+      ```bash
+      tar xvf kots_darwin_all.tar.gz
+      ```
+   * **Linux (AMD)**:
+
+      ```bash
+      tar xvf kots_linux_amd64.tar.gz
+      ```
+   * **Linux (ARM)**: Download
```
[the remark this answers is not in the document]
Comment (paigecalvert) [reply docs/reference/kots-cli-getting-started.md:128]: thank you for catching that!

**Pull request merged**